src/SmartComponents/HybridInventoryTabs/ImmutableDevices.test.js

```diff
@@ -1,34 +1,115 @@
 import React from 'react';
 import '@testing-library/jest-dom';
-import { renderWithContext } from '../../Utilities/TestingUtilities';
+import { screen, waitFor } from '@testing-library/react';
+import { ComponentWithContext } from '../../Utilities/TestingUtilities';
 import ImmutableDevices from './ImmutableDevices';
+import { render } from '@testing-library/react';
+import AsynComponent from '@redhat-cloud-services/frontend-components/AsyncComponent';
+import { useGetEntities } from './helpers';
+
+jest.mock('@redhat-cloud-services/frontend-components/AsyncComponent', () => ({
```

Comment:
I had to mock it here as well to be able to test AsyncComponent calls. Otherwise I was getting **Matcher error: received value must be a mock or spy function**